plugin/aws/s3Object.go

```diff
@@ -39,8 +39,9 @@ func newS3Object(o *s3Client.Object, name string, bucket string, key string, cli
 	mtime := awsSDK.TimeValue(o.LastModified)
 	s3Obj.
 		Attributes().
-		SetCtime(mtime).
+		SetCrtime(mtime).
```

Comment:
This seems like it should be something else. Does S3 return anything besides LastModified (which definitely corresponds to mtime)?

Reply:
Unfortunately no -- see https://github.com/puppetlabs/wash/blob/master/plugin/aws/s3Object.go#L32-L35